packages/server/lib/project-base.ts

```diff
@@ -160,42 +154,36 @@ export class ProjectBase extends EE {
 
     this._server = new ServerBase(cfg)
 
-    let studioManager: StudioManager | null
+    const studioLifecycleManager = new StudioLifecycleManager()
 
-    if (process.env.CYPRESS_ENABLE_CLOUD_STUDIO || process.env.CYPRESS_LOCAL_STUDIO_PATH) {
-      studioManager = await getAndInitializeStudioManager({
-        projectId: cfg.projectId,
-        cloudDataSource: this.ctx.cloud,
-      })
+    void studioLifecycleManager.initializeStudioManager({
```

Comment:
As I'm thinking about this, I believe this function gets called in run mode. So we should only do this:

```js
if (!cfg.isTextTerminal) {}
```